From a34e5fcaac3ba796f34cdeb78436daea3b0a55ab Mon Sep 17 00:00:00 2001
From: Naveen Saini <naveen.kumar.saini@intel.com>
Date: Wed, 14 Jun 2023 10:49:58 +0800
Subject: [PATCH] Execute pre_build_check during hypervisor build is causing
 error

| make: build//hypervisor/hv_prebuild_check.out: Command not found

Build and execute as native.

Upstream-Status: Inappropriate

Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
---
 hypervisor/Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/hypervisor/Makefile b/hypervisor/Makefile
index 029b4cfb4..c1038ec08 100644
--- a/hypervisor/Makefile
+++ b/hypervisor/Makefile
@@ -424,9 +424,6 @@ $(error Please either install "iasl" or provide the path to "iasl" by using the
 endif
 
 $(PRE_BUILD_CHECKER): $(HV_CONFIG_H) $(HV_CONFIG_TIMESTAMP)
-	@echo "Start pre-build static check ..."
-	$(MAKE) -C $(PRE_BUILD_DIR) BOARD=$(BOARD) SCENARIO=$(SCENARIO) CHECKER_OUT=$(PRE_BUILD_CHECKER)
-	@$(PRE_BUILD_CHECKER)
 
 $(HV_ACPI_TABLE_TIMESTAMP): $(HV_CONFIG_TIMESTAMP)
 	@echo "generate the binary of ACPI tables for pre-launched VMs ..."
-- 
2.34.1

